features/modifying_order_total.feature

```diff
@@ -0,0 +1,36 @@
+@managing_orders
+Feature: Modifying order total
```

Comment:
What about custom decimal amount and testing decimals containing three or more decimal places?

Reply:
We can test decimal totals, for sure, but about "three or more decimal places" - I don't know is it crucial for now, it does not provide any valid value imo

Reply:
ok then